scripts/install-snapshot.sh

```diff
@@ -2,9 +2,11 @@
 
 # This script can be used to install/delete snapshotcontroller and snapshot beta CRD
 
-SNAPSHOT_VERSION=${SNAPSHOT_VERSION:-"v2.1.1"}
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
```

Comment:
Don't use `${BASH_SOURCE[0]}`, just use `${0}` like done in other scripts.

Reply:
Most of the scripts still use BASH_SOURCE,

```
[0] root 😎 04:20:27📂ceph-csi✨ cd examples/rbd/
[0] root 😎 04:20:36📂rbd✨ cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd
/home/pkalever/work/contributions/ceph-csi/examples/rbd
[0] root 😎 04:20:42📂rbd✨ cd "$(dirname "${0}")" >/dev/null 2>&1 && pwd
dirname: invalid option -- 'b'
Try 'dirname --help' for more information.
/home/pkalever/work/contributions/ceph-csi/examples/rbd

```

Using "${0}" seems to be erroring in the above case.

Reply:
please ignore my comments above, I missed out on something and now I understand it.